src/ResourceManager/Automation/Commands.Automation/Common/AutomationClientDSC.cs

```diff
@@ -1238,6 +1240,7 @@ public IEnumerable<Model.NodeConfiguration> ListNodeConfigurations(string resour
                     automationAccountName,
                     nodeConfigurationName,
                     null);
+
                 if (nodeConfigurationModel != null)
                 {
                     if (!overWrite)
```

Comment:
@vrdmr the block below is where the use of `Force` is incorrect. Rather than throwing after determining `Force` was not provided, you should prompt the user if they would like to continue using `ShouldContinue`.